source/includes/_webhook_integration.md

```diff
@@ -2,18 +2,30 @@
 
 ## Client Leads Webhook
 
-Evocalize can notify your organization via webhook any time a new lead is ingested from an upstream platform such as Facebook or Google. Upon notification from the upstream platform that new lead data has been created, we combine that with Evocalize data and your data, and send it to you, allowing you to handle leads in near real-time.
+Evocalize can notify your organization via webhook any time a new lead is ingested from an upstream platform such as
+Facebook or Google. Upon notification from the upstream platform that new lead data has been created, we combine that
+with Evocalize data and your data, and send it to you, allowing you to handle leads in near real-time.
 
 ### Setup
 
-You will need to work with your account manager and supply Evocalize with an endpoint url where you would like to be notified with the payload specified below. Your endpoint url must use SSL (https) and accept a POST. You will need to have a `ClientKeyID` and a `ClientSecret`. These are the same values that you will use to call our API for other operations.
+You will need to work with your account manager and supply Evocalize with an endpoint url where you would like to be
+notified with the payload specified below. Your endpoint url must use SSL (https) and accept a POST. You will need to
+have a `ClientKeyID` and a `ClientSecret`. These are the same values that you will use to call our API for other
+operations.
 
 ### Policies
 
-- Each lead may be delivered more than once. This is rare, but it is possible. The leads[].id is unique to a particular lead and will not change between deliveries if a lead is delivered multiple times.
-- Any response code in the range of 200 - 299 is considered a success, any other response code is considered a failure and we will attempt to redeliver the lead notification.
-- The HTTPS request will time out after 5 seconds. If this happens, delivery is considered a failure and we will attempt to redeliver the lead notification.
-- The time between retry attempts is exponential, with a maximum of 15 minutes between tries. After three hours have elapsed since the first attempt, we will no longer try to deliver the leads. If something has happened causing your platform to stop ingesting leads for more than 3 hours, contact your Evocalize account representative.
+- ℹ️ **Duplicate Leads: If any update sent to your server fails, we will retry several times with decreasing frequency until
+  we get a success. While this helps ensure you receive all of your leads, it can create duplicates on your end so your
```

Comment:
This isn't the only thing that can cause duplicates. Facebook also delivers duplicates to us sometimes

Reply:
I'll add a bit on marketing channels also sending duplicates

Reply:
Yeah - I think keeping this simple rather than spelling out every single case is the key. Also leaving off "Facebook could send us duplicate leads, which in turn means we could send you duplicates because they sent us duplicates" is overkill and opens the door for a partner to ask, "well, why don't you just dedupe on your side to prevent us from having to" - which we then have to answer, "well we can do that but it's still possible that you'll get duplicates due to retry logic in the event that sending you a lead looks like a failure to us due to a timeout but you actually received it" <-- all of that is way too long and I don't want to have that discussion with partners every time they read it 😆

Reply:
ah true, that makes sense